Src/Eiffel/interface/new_graphical/stones/local_stone.e

```diff
@@ -0,0 +1,206 @@
+note
+	description	: "Stone representing an eiffel feature local stone."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	LOCAL_STONE
+
+inherit
+	FEATURE_STONE
```

Comment:
inheriting from AST_STONE is to know the location of the variable, thanks to the ast: AST_EIFFEL from AST_STONE.
With FEATURE_STONE, we would just know where is the feature declaration.